lib/blacklight_cornell/virtual_browse.rb

```diff
@@ -201,28 +201,30 @@ def callnumber_cleanup(callnumber)
     callnumber.gsub("Oversize ","").gsub("Rare Books ","").gsub("ONLINE ","").gsub("Human Sexuality ","").gsub("Ellis ","").gsub("New & Noteworthy Books ","").gsub("A.D. White Oversize ","").sub("+ ","")
   end
 
-  def get_googlebooks_image(oclc, isbn, format)
-    uri = URI("https://books.google.com/books")
-    if oclc.present? and !oclc.include?("not found")
-      params = { :bibkeys => "OCLC:#{oclc}", :jscmd => "viewapi", "callback" => "?"}
-      uri.query = URI.encode_www_form(params)
-      result = Net::HTTP.get(uri)
-      result = eval(result.gsub("var _GBSBookInfo = ",""))
-      if result.present? && result.values[0].present? && result.values[0][:thumbnail_url].present?
-        return result.values[0][:thumbnail_url]
-      end
-    end
-    if isbn.present? and !isbn.include?("not found")
-      params = { :bibkeys => "OCLC:#{isbn}", :jscmd => "viewapi", "callback" => "?"}
```

Comment:
Hmm, strange that isbn also generates a bibkey with the "OCLC:" prefix, but if that's intentional, it's great that you DRY-ed up this code.